crates/environ/src/fact/trampoline.rs

```diff
@@ -78,52 +78,69 @@ pub(super) fn compile(module: &mut Module<'_>, adapter: &AdapterData) {
         free_locals: HashMap::new(),
         traps: Vec::new(),
         result,
-        top_level_translate: true,
+        fuel: INITIAL_FUEL,
     }
     .compile_adapter(adapter, &lower_sig, &lift_sig)
 }
 
-/// Compiles a helper function which is used to translate `src` to `dst`
-/// in-memory.
-///
-/// The generated function takes two arguments: the source pointer and
-/// destination pointer. The conversion operation is configured by the
-/// `src_opts` and `dst_opts` specified as well.
-pub(super) fn compile_translate_mem(
-    module: &mut Module<'_>,
-    result: FunctionId,
-    src: InterfaceType,
-    src_opts: &Options,
-    dst: InterfaceType,
-    dst_opts: &Options,
-) {
+/// TODO
```

Comment:
todo

Reply:
Oops this was actually intended for a separate PR so I'm going to remove this